client-spark/spark-3-columnar-shuffle/src/main/java/org/apache/spark/sql/execution/columnar/CelebornColumnDictionary.java

```diff
@@ -58,6 +60,6 @@ public double decodeToDouble(int id) {
 
   @Override
   public byte[] decodeToBinary(int id) {
-    return stringDictionary[id].getBytes();
+    return stringDictionary[id].getBytes(UTF_8);
```

Comment:
Avoid static import unless it has real improvement for code style
```suggestion
    return stringDictionary[id].getBytes(StandardCharsets.UTF_8);
```